packages/m/materialstyle.json

```
@@ -0,0 +1,51 @@
+{
+  "description": "Material Style is a Material Design UI library based on Bootstrap.",
+  "filename": "js/materialstyle.min.js",
+  "homepage": "https://materialstyle.github.io/",
+  "keywords": [
+    "materialstyle",
+    "material-style",
+    "material-design",
+    "css",
+    "sass",
+    "scss",
+    "mobile-first",
+    "responsive",
+    "front-end",
+    "framework",
+    "web",
+    "bootstrap",
+    "bootstrap-material-design",
+    "bootstrap-theme",
+    "google-material-design"
+  ],
+  "license": "MIT",
+  "name": "@materialstyle/materialstyle",
```

```suggestion
  "name": "materialstyle",
```
This should match the filename